frontend/src/views/app/Search.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <!-- eslint-disable max-len -->
+  <div class="mx-4 sm:mx-16 lg:mx-32">
+    <NavBar v-bind:currentRoute="'Search'"></NavBar>
+    <section v-if="searchSubmitted">results</section>
+    <section v-if="!searchSubmitted" class="flex flex-col my-8 md:my-12">
+        <div>
+          <FilterSlider
+            v-bind:min="18"
+            v-bind:max="100"
+            v-bind:name="'age'"
+            v-on:saveFilter="saveFilter"></FilterSlider>
+          <FilterSlider
+            v-bind:min="18"
+            v-bind:max="100"
+            v-bind:name="'distance'"
+            v-on:saveFilter="saveFilter"></FilterSlider>
+          <FilterSlider
+            v-bind:min="18"
+            v-bind:max="100"
+            v-bind:name="'popularity'"
+            v-on:saveFilter="saveFilter"></FilterSlider>
+        </div>
+        <div>
+          <MultipleFilters
+            v-bind:options="[
+            'swimming', 'wine', 'reading', 'foodie', 'netflix', 'music', 'yoga', 'golf',
+            'photography', 'baking', 'shopping', 'outdoors', 'art', 'travel', 'hiking',
+            'running', 'volunteering', 'cycling', 'climbing', 'tea', 'fishing', 'soccer',
+            'museum', 'dancing', 'surfing', 'karaoke', 'parties', 'diy',
+            'walking', 'cat lover', 'movies', 'gardening', 'trivia', 'working out',
+            'cooking', 'gamer', 'brunch', 'blogging', 'picknicking', 'athlete',
+            'dog lover', 'politics', 'environmentalism', 'instagram', 'spirituality',
+            'language exchange', 'sports', 'comedy', 'fashion', 'disney', 'vlogging',
+            'astrology', 'board games', 'craft beer', 'coffee', 'writer',
+          ]"
```

Comment:
Should I add a route that serves the list of tags ? So we can modify it easily if needed in the backend and your frontend can just get a public list of tags